binds/python/bind_immutable.cpp

```diff
@@ -96,6 +96,7 @@ static void bind_immutable_module(py::module &m) {
                     return py::make_iterator(morpho->depth_begin(), morpho->depth_end());
                 case morphio::IterType::BREADTH_FIRST:
                     return py::make_iterator(morpho->breadth_begin(), morpho->breadth_end());
+                case morphio::IterType::UPSTREAM:
```

Comment:
Why don't you let the default handle it anyway ?

Reply:
Is clang complaining for not all enum values being covered ?

Reply:
yeah, and it's better to be explicit about what is being skipped - one can't keep all the enum values in their head when they are editing a switch statement, so this way it's obvios what is happening